### ISSUE TITLE
More examples of incorrect nomenclature

### ISSUE THREAD

**Issue by arouinfar** [phetsims/build-a-molecule#153]
In #149 we discovered that the database had an incorrect entry for ethenone. Since that issue contained some tangential discussion, @Denz1994 and I decided to open a new issue to document other examples. There are more than 9000 named molecules in [otherMoleculesData.js](https://github.com/phetsims/build-a-molecule/blob/master/js/common/model/data/otherMoleculesData.js), and it's unreasonable to check them all by hand. However, a good rule of thumb would be for QA to cross-reference the [pubchem database](https://pubchem.ncbi.nlm.nih.gov/) whenever they encounter a named molecule they are unfamiliar with.

This structure is not butane (C4H10). @Denz1994 confirmed there is error in otherMoleculeData.js which begins with `butane|C4H9|137553|...`
<img src="https://user-images.githubusercontent.com/8419308/73396014-ce7fc480-429d-11ea-8b7f-09df6d5cb2da.png" width=250>
(As a side note the 3rd datapoint in the entry is the PubChem CID, which reveals this entry belongs to [2-butyl radical](https://pubchem.ncbi.nlm.nih.gov/compound/137553). This echos a bit of #150).

**Comment by Denz1994** [comment]
The entry for `C4H9|Butane` was removed. The PubChem entry for [Butane](https://pubchem.ncbi.nlm.nih.gov/compound/7843) includes an alternate listing for Butane as `C4H10|n-Butane`, which is currently being used.

I think #148 needs to be fixed before we can create it though. I was only able to create isobutane. 
![image](https://user-images.githubusercontent.com/13894380/73418245-d9ac1200-42e8-11ea-98fe-041723646120.png)

**Comment by Denz1994** [comment]
Also worth noting, the full source of molecules can be found here:
[Collection Molecules](https://raw.githubusercontent.com/phetsims/build-a-molecule/master/js/common/model/data/collectionMoleculesData.js) and [Other Molecules](https://raw.githubusercontent.com/phetsims/build-a-molecule/master/js/common/model/data/otherMoleculesData.js)

The data is formated so it is read like so:
`moleculeName|generalFormula|compoundCID|2d3dOrBoth|totalAtoms|totalBonds|{Element n xyz}`

**Comment by arouinfar** [comment]
@Denz1994 we should have two entries for C4H10: `n-butane` and `isobutane`. The Java behavior currently handles both cases correctly. The C4H9 entry for `butane` is the incorrect one.

**Comment by Denz1994** [comment]
> @Denz1994 we should have two entries for C4H10: n-butane and isobutane. The Java sim currently handles both cases correctly. The C4H9 entry for butane is the incorrect one.

Confirmed in `otherMoleculesData.js` we have both `n-butane` and `isobutane`.

**Comment by Denz1994** [comment]
It was discussed during the design meeting on 01/31/20:

We haven't gotten reports of problematic molecules. We should leave up to users to report on any molecules that are incorrect. QA shouldn't have to check against each molecule for its existence.

**Comment by Denz1994** [comment]
@ariel-phet mentioned: 

> If QA finds an obvious problem with the name of a molecule that is built during testing, then it should be reported. But other than that we will defer to user reports.

I will close this issue and any erroneous molecules that are noted during testing a separate issue should be opened. Closing this issue.
